.github/workflows/rebase.yml

```diff
@@ -17,6 +17,16 @@ jobs:
     if: github.event.issue.pull_request != '' && github.event.comment.body == 'GitHub, rebase'
     runs-on: ubuntu-latest
     steps:
+      - uses: khan/pull-request-comment-trigger@master
+        name: React with rocket on run
+        with:
+          trigger: 'GitHub, rebase'
+          reaction: rocket
+          prefix_only: true
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+      - run: 'echo We print it here for this action to work'
+        if: steps.check.outputs.triggered == 'true'
```

Comment:
we can make it:
```
run: 'echo We print it here for this action to work'
         if: 'true'
```

as we always has to execute it.

Reply:
Solved.